packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/functionConfiguration.bicep

```diff
@@ -0,0 +1,81 @@
+param functionAppName string
```

Comment:
This `functionConfiguration.bicep` file was removed from dev branch, is it needed any more?

Reply:
Good catch! These test files should be removed together.

Reply:
Removed.